src/pages/Projects.jsx

```diff
@@ -61,18 +62,14 @@ const ProjectTitle = ({ index, title, logo, links }) => {
           sx={{
             objectFit: 'fill',
             maxWidth: '80%',
-            height: logo === '/assets/codePdx/pass-logo.svg' ? 'auto' : '150px',
-            pt: logo != '/assets/codePdx/logo-record-sponge-oregon.svg' ? '5%' : 'none'
+            height: { xs: '55px', sm: '100px' }, //55 logo === '/assets/codePdx/pass-logo.svg' ? 'auto' : '150px'
+            // pt: logo != '/assets/codePdx/logo-record-sponge-oregon.svg' ? '5%' : 'none'
```

Comment:
If we go with these changes, should remove the commented-out code

Reply:
I removed these comments and merged.